### PR TITLE
Newline between license and package.

### PR DESCRIPTION
**Pull request by anuraaga**
I noticed `opentelemetry-java` has a newline between license and package, but not here. It seems like being consistent would be nice since users should have their IDE setup in the same way for both. From what I know, IntelliJ always adds the newline no matter what so it's a bit annoying when the build breaks because of that.

Just a strawmans proposal happy to close if not a good idea.

### CODE REVIEW

**Comment by iNikem** [comment]
If we think this is important, then either google-java-format or (preferably) spotless must enforce it.

**Comment by anuraaga** [comment]
Yeah spotless is, I just added a new line to the current file. There's no way I'd do this by hand :)

File: gradle/enforcement/spotless.license.java [hunk 1]
```diff
@@ -13,3 +13,4 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
+
```
Comment (iNikem) [inline gradle/enforcement/spotless.license.java:16]: Strange. There is new line at the end of this file in my clone...

Comment (anuraaga) [reply gradle/enforcement/spotless.license.java:16]: Yeah spotless seems to ignore one trailing line so adding a second did the trick

**Pull request merged**